lock-api/src/main/metrics/metric-schema.yml

```diff
@@ -30,4 +30,10 @@ namespaces:
         type: timer
         docs: observed call duration during leader election
 
+  timestampLease:
+    docs: Timestamp lease client side telemetry
+    metrics:
+      notEnoughFreshTimestamps:
```

Comment:
pedantry: I'd probably have `instances` somewhere in the name, to be explicit that it's not the amount of shortfall that matters in this count